programs/pluto/kernel_xfrm.c

```diff
@@ -387,6 +400,36 @@ static void kernel_xfrm_flush(struct logger *logger)
 			  &recv_errno, logger);
 }
 
+static void print_ext_ack(const struct nlmsghdr *n, struct jambuf *buf)
```

Comment:
Can you please make the signature `jam_ext_ack(struct jambuf *, const struct nlmsghdr *)` so it's like the other jam functions.

Reply:
never mind, I noticed something else